### PR TITLE
ROE-1815 updating beneficial-owner-statements page content

### PR DESCRIPTION
**Pull request by SVarallo**
### JIRA link
https://companieshouse.atlassian.net/browse/ROE-1815

### Change description
Updating the content on the beneficial-owner-statements page to bring it up to the latest version on the prototype

### Work checklist

- [ ] Tests added where applicable
- [x] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

File: views/beneficial-owner-statements.html [hunk 1]
```diff
@@ -43,15 +35,15 @@ <h1 class="govuk-heading-xl">{{ title }}</h1>
           items: [
             {
               value: "ALL_IDENTIFIED_ALL_DETAILS",
-              text: "All beneficial owners have been identified and all required information can be provided",
+              text: "All beneficial owners have been identified and I can provide all the required information",
```
Comment (mouhajer-ch) [inline views/beneficial-owner-statements.html:38]: We need to make changes on check your answers page [here](https://github.com/companieshouse/overseas-entities-web/blob/main/views/includes/check-your-answers/beneficial-owner-statements.html) as well.

Could you please add a few text check on related unit tests, thanks Sam

File: test/__mocks__/text.mock.ts [hunk 2]
```diff
@@ -9,7 +9,7 @@ export const BENEFICIAL_OWNER_GOV_PAGE_HEADING = "Tell us about the government o
 export const BENEFICIAL_OWNER_OTHER_PAGE_HEADING = "Tell us about the other legal entity";
 export const BENEFICIAL_OWNER_STATEMENTS_PAGE_REDIRECT = "Found. Redirecting to /register-an-overseas-entity/beneficial-owner-statements";
 export const BENEFICIAL_OWNER_TYPE_PAGE_REDIRECT = "Found. Redirecting to /register-an-overseas-entity/beneficial-owner-type";
-export const BENEFICIAL_OWNER_STATEMENTS_PAGE_HEADING = "Have you identified any beneficial owners?";
```
Comment (mattch1) [inline test/__mocks__/text.mock.ts:12]: Looks fine, just wondering why the mock did not match the title before as they it does now.

File: views/beneficial-owner-statements.html (hunk 1 above)
Comment (SVarallo) [reply views/beneficial-owner-statements.html:38]: Ok, I can do that

File: test/__mocks__/text.mock.ts (hunk 2 above)
Comment (SVarallo) [reply test/__mocks__/text.mock.ts:12]: True it's an odd one, if I change the unit tests to something else however it does fail

Comment (mouhajer-ch) [reply test/__mocks__/text.mock.ts:12]: Its referring to the hidden field [here](https://github.com/companieshouse/overseas-entities-web/blob/f7f6f8517b50738b90679e0a58b83ef3ea9f8143/views/beneficial-owner-statements.html#L38) that I think should be modified as well, text should be `Has the overseas entity identified any registrable beneficial owners?`, same as the title.

Comment (mouhajer-ch) [reply test/__mocks__/text.mock.ts:12]: Better if you use the title variable:
```
...
  legend: {
    text: title,
    isPageHeading: false,
    classes: "govuk-visually-hidden"
  }
...
```

**Review by mouhajer-ch — APPROVED** [review]

**Pull request merged**